lib/token.h

```diff
@@ -1554,7 +1554,7 @@ class CPPCHECKLIB Token {
 
     void printAst(bool verbose, bool xml, const std::vector<std::string> &fileNames, std::ostream &out) const;
 
-    void printValueFlow(bool xml, std::ostream &out) const;
+    void printValueFlow(const std::vector<std::string>& files, bool xml, std::ostream &out) const;
```

Comment:
I don't follow why we don't use mList.getFiles() in the method?

Reply:
The calling code uses the `TokenList` object to call a `Token` object which calls back into the `TokenList`. This change removed that (see also above).

Also it matches what we did in `printAst()` and it would be preferable to not have any knowledge about the list in the Token itself (the last remaining usage is the keyword check).